README.md

```diff
@@ -22,82 +23,108 @@ Ask Joanna on Slack to set you up, you'll get an email invitation to download Te
+3. Go to the following link: https://reactnative.dev/docs/environment-setup - choose the 'React Native CLI Quickstart' tab, and follow the instructions for your platform
+
+   > **Note** Where the instructions refer to a particular version of Android or the Android SDK platform, use **Android 10** and **SDK platform 29** instead of the more recent one.
+
+   > **Note** You can skip the sections of the React Native docs that describe setting up and running a new project, since we already have one.
 
 # Setup and first run
 
-1. Ensure that you've gone through the following link for your particular platform: https://reactnative.dev/docs/environment-setup
+1. Ensure that you've gone through the setup instructions at the following link for your particular platform: https://reactnative.dev/docs/environment-setup (see notes [above](#requirements-to-run-the-project))
+
+2. Clone the code from GitHub:
 
-2. Pull the code from Git
+   > `git clone https://github.com/Scottish-Tech-Army/Volunteer-app.git`
 
 3. Open Command terminal
 
-4. Go to the `api` folder inside the project folder (e.g. **\Volunteer-app path\api**)
+4. Go to the `api` folder inside the project folder (e.g. **/path/to/Volunteer-app/api**)
 
-5. Copy the `.env.example` file in the api root folder and name your new file `.env` in the same folder. Fill in the empty values (`""`) in your file for any credentials/settings (API keys for STA Jira API access, AirTable, etc).
-    >**Note** Credentials themselves not provided, these should be requested/provided on joining the dev group.
+5. Copy the `.env.example` file in the api root folder and name your new file `.env` in the same folder. Fill in the empty values (`""`) in your file for any credentials/settings (API keys for STA Jira API access, AirTable, etc)
 
-6. At the command prompt type `npm install` then `npm start` to start the Volunteer App API server.
+   > **Note** For security reasons, the credentials themselves are not provided here. Ask in the [volunteer-app](https://scottishtecharmy.slack.com/archives/C01SUL6K5E1) Slack channel when you join the dev group, and somebody will send them to you.
 
-7. Go to the `app` folder inside the project folder (e.g. **\Volunteer-app path\app**)
+6. At the command prompt type `npm install` then `npm start` to start the Volunteer App API server. You should see a message that says `Volunteer App API is listening on port <number>` - make a note of the port number for later
+
+7. Go to the `app` folder inside the project folder (e.g. **/path/to/Volunteer-app/app**)
 
 8. At the command prompt type `npm install`
-    >**Note:** Inside the `app` folder there is package-lock.json. Everytime this is modified, it is advised to repeat step 6 before  running the project.
 
-    >**Note:** you may need to run `npm install --legacy-peer-deps`
+   > **Note** Inside the `app` folder there is `package-lock.json`. Everytime this is modified, it is advised to repeat step 6 before running the project.
```

Comment:
Reading this again, wonder should this line say repeat this step (rather than step 6) -- and every time `package.json` or `package-lock.json` is modified?

And maybe we should say something similar above on step 6 itself for the `api` folder?